docs/user-guides/devcontainers/index.md

```diff
@@ -38,62 +25,64 @@ which allows for extensive customization of your development setup.
 When a workspace with the dev containers integration starts:
 
 1. The workspace initializes the Docker environment.
-1. The integration detects repositories with a `.devcontainer` directory or a
-   `devcontainer.json` file.
-1. The integration builds and starts the dev container based on the
-   configuration.
+1. The integration detects repositories with a `.devcontainer` directory or a `devcontainer.json` file.
+1. The integration builds (or rebuilds) and starts the dev container based on the configuration.
 1. Your workspace automatically detects the running dev container.
+1. If the configuration changes, the workspace prompts you to rebuild the dev container.
 
 ## Features
 
-### Available Now
+### Detection & Lifecycle
 
-- Automatic dev container detection from repositories
-- Seamless dev container startup during workspace initialization
-- Integrated IDE experience in dev containers with VS Code
-- Direct service access in dev containers
-- Limited SSH access to dev containers
+- Automatic discovery of `.devcontainer` configs.
+- Change detection with rebuild prompts.
+- Rebuild containers with one click from the Coder dashboard or from the CLI.
 
-### Coming Soon
+### Connectivity
 
-- Dev container change detection
-- On-demand dev container recreation
-- Support for automatic port forwarding inside the container
-- Full native SSH support to dev containers
+- Full SSH access directly into dev containers (`coder ssh --container ...`).
+- Automatic port forwarding based on `appPort`, `forwardPorts`, or `docker-compose.yml`.
```

Comment:
We don't officially support `docker-compose.yaml` yet. I suggest we rewrite this entire line as just:

```suggestion
- Automatic port forwarding.
```

We don't use `forwardPorts` currently (mainly useful to connect to other containers defined in docker compose), and `appPort`s show up in the UI as a "coder app" for the devcontainer. Services listening on ports are surfaced the same way as for regular workspaces. I think defining all these behaviors would be more confusing than seeing how customers use this and how it matches/breaks their expectations.